Samples/Excel.OfflineStorageAddin/README.md

```diff
@@ -2,16 +2,17 @@
 page_type: sample
 urlFragment: office-add-in-access-data-offline
 products:
-- office-excel
-- office-365
-- office-powerpoint
-- office-word
+  - office
+  - office-excel
+  - office-365

```

Comment:
let's remove office-365 (here and other instances) since Office 365 is an obsolete term. office and office-{application} roll up to m365. Update any internal guidance accordingly

```suggestion
```

Reply:
Removed here and in other instances.